chainerx_cc/chainerx/scalar.h

```diff
@@ -104,6 +104,13 @@ class Scalar {
         return this->UnwrapAndCast<int64_t>() + rhs.UnwrapAndCast<int64_t>();
     }
 
+    Scalar operator-(const Scalar& rhs) const {
```

Comment:
Introduced for this computation https://github.com/chainer/chainer/pull/6496/files#diff-82d97e7f86c1e4c6e00277b3f56cbfd8R918